### DISCUSSION TITLE
Repeatable support dependent field (dependsOn) functionality.

### DISCUSSION THREAD

**Discussion by heshammostafa123** [laravel/nova-issues#6130] [Ideas & Feature Requests]
Repeatable support dependent field (dependsOn) functionality.

**Comment by AbdallahAli912** [comment]
plz add  support for dependent field  in  Repeatable fields.

**Comment by heshammostafa123** [comment]
@davidhemphill @crynobone

**Reply by davidhemphill** [reply]
Please don't spam tag maintainers. We regularly look through feature requests. 😉

**Comment by Denobraz** [comment]
@heshammostafa123 maybe you find any solution for this problem?

**Comment by Denobraz** [comment]
Please, add support for this feature!

**Comment by webard** [comment]
It would be great if "Repeater Fields" feature got out of Beta stage :)

**Comment by ghost** [comment]
This functionality is still very much needed.

**Comment by maul75** [comment]
Please implement as soon as possible. Make life much easier ...
